roles/ciao-compute/README.md

```diff
@@ -1,33 +1,36 @@
-ciao-compute
-=========
+# ciao-compute
 Ansible role to install the compute node for a CIAO cluster
 
-Requirements
-------------
-None
+## Requirements
+- A previously setup of a CIAO controller node.
+  You can setup one with `ciao-controller` role
 
-Role Variables
---------------
-* `fedora_cloud_image_url`
-* `clear_cloud_image_url`
+## Role Variables
+The available variables for this roles are the variables from `ciao-common` plus the following
 
-Dependencies
-------------
-* `ciao-controller`
+Variable  | Default Value | Description
+--------  | ------------- | -----------
+fedora_cloud_image_url | [Fedora-Cloud-Base-23-20151030.x86_64.qcow2](https://dl.fedoraproject.org/pub/fedora/linux/releases/23/Cloud/x86_64/Images/Fedora-Cloud-Base-23-20151030.x86_64.qcow2) | URL for the latest fedora cloud image
+clear_cloud_image_url | [clear-8740-cloud.img.xz](https://download.clearlinux.org/image/clear-8740-cloud.img.xz) | URL for the latest clearlinux cloud image
 
```

Comment:
Its an example, but the default value should work too.
I will update with the latest from today